frontend/src/components/users/GeneralInfo.vue

```diff
@@ -1,129 +1,126 @@
-                                <i class="fa-2x fab fa-twitter-square" />
-                            </a>
-                            <a
-                                v-if="userProfile.instagram_url"
-                                v-show="userProfile.instagram_url"
-                                :href="userProfile.instagram_url"
-                                class="instagram-link"
-                                target="_blank"
-                            >
-                                <i class="fa-2x fab fa-instagram" />
-                            </a>
+                    <div class="column user-info-stats">
+
+                        <div class="user-stats-title">
+                            <h1 class="subtitle is-3">{{ fullName }}</h1>
+                            <div class="user-social">
+                                <a
+                                    v-if="userProfile.facebook_url"
+                                    v-show="userProfile.facebook_url"
+                                    :href="userProfile.facebook_url"
+                                    class="facebbok-link"
+                                    target="_blank"
+                                >
+                                    <i class="fa-2x fab fa-facebook-square" />
+                                </a>
+                                <a
+                                    v-if="userProfile.twitter_url"
+                                    v-show="userProfile.twitter_url"
+                                    :href="userProfile.twitter_url"
+                                    class="twitter-link"
+                                    target="_blank"
+                                >
+                                    <i class="fa-2x fab fa-twitter-square" />
+                                </a>
+                                <a
+                                    v-if="userProfile.instagram_url"
+                                    v-show="userProfile.instagram_url"
+                                    :href="userProfile.instagram_url"
+                                    class="instagram-link"
+                                    target="_blank"
+                                >
+                                    <i class="fa-2x fab fa-instagram" />
+                                </a>
+                            </div>
                         </div>
-                    </div>
 
-                    <div class="user-stats-contact">
-                        <span>Odessa</span>
-                    </div>
+                        <div class="user-stats-contact">
+                            <span>Odessa</span>
+                        </div>
 
-                    <div class="user-stats-complaint">
-                        <span>Пожаловаться на этого человека?</span>
+                        <div class="user-stats-complaint">
+                            <span>Пожаловаться на этого человека?</span>
```

Comment:
Use localization for constant phrases

Reply:
This is removed here: https://github.com/BinaryStudioAcademy/bsa-2018-hedonist/pull/583